src/components/Avatar.js

```diff
@@ -55,15 +59,15 @@ const defaultProps = {
     iconAdditionalStyles: [],
     containerStyles: [],
     size: CONST.AVATAR_SIZE.DEFAULT,
-    fill: themeColors.icon,
+    fill: null,
```

Comment:
Why is the default fill color set to null here instead of the icon theme color?

Reply:
When we set the default value of fill property to `themeColors.icon` it overrides default color of svg icons. It leads to situations when we are unable to display the defaultAvatar component correctly. Passing null value to the fill property indicates that we want to use default fill color of svg icons. The app looks like this when we set the fill property value to `themeColors.icon`

![simulator_screenshot_A3E79E43-5243-4C6E-B329-04CA61071F76](https://github.com/Expensify/App/assets/47774969/db66e5ac-bc6f-4ac6-ad94-124c44d4ce9e)